src/util/findCodeblockFromChannel.ts

```diff
@@ -8,7 +8,7 @@ export const PLAYGROUND_REGEX = /https?:\/\/(?:www\.)?typescriptlang\.org\/(?:pl
 export async function findCodeblockFromChannel(
 	channel: TextChannel,
 	ignoreLatest?: boolean,
-) {
+): Promise<string | undefined> {
```

Comment:
Is there any reason why you made this type explicit?

Reply:
The inferred type was just `Promise<string>` because of the usage of `.filter()[0]` to return `undefined` when no code block is found

Reply:
Looks like this will probably be made redundant by [microsoft/TypeScript#39560](https://github.com/microsoft/TypeScript/pull/39560) once a release comes out with that PR in.